src/app/components/PlanshipProvider.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
+import { withPlanshipCustomerProvider } from '@planship/planship-react';
+import { getAccessToken } from '../lib/planship';
+import { CustomerSubscriptionWithPlan, JSONValue } from '@planship/fetch';
+import { useCurrentUser } from './CurrentUserProvider';
+
+export default function PlanshipProvider({
+  children,
+  initialEntitlements,
+  initialSubscrptions,
+}: {
+  children: React.ReactNode;
+  initialEntitlements: JSONValue;
+  initialSubscrptions: CustomerSubscriptionWithPlan[];
+}) {
+  const PlanshipCustomerProvider = withPlanshipCustomerProvider(
+    {
+      url: 'http://localhost:8002',
```

Comment:
I think I brought this up in the Nuxt example, but these ports should be configurable as 800x are so common.

Reply:
This should actually use env vars, fixed.